library/auth.inc

```diff
@@ -90,12 +90,7 @@ if (
     authLoginScreen(true);
 } else {
     // Check if session is valid (already logged in user)
-    if (AuthUtils::authCheckSession()) {
-        // Session is valid
-        if (isset($_SESSION['pid']) && empty($GLOBALS['DAEMON_FLAG'])) {
-            require_once("{$GLOBALS['srcdir']}/patient.inc");
```

Comment:
hi @sunsetsystems, looks like we're going to need to add this somewhere in here or the calling scripts, [like here](https://github.com/openemr/openemr/blob/master/interface/forms/newpatient/new.php#L15)

Reply:
just added it to `forms/new.php` in pending [PR](https://github.com/openemr/openemr/pull/4436/commits/014f9772021c392df21ed6e4d8ffcf498070793a)

Reply:
Thanks @stephenwaite for the fix. That require_once was left over from when auth.inc logged patient info here, but it doesn't any more so this was clearly the wrong place for it.

Reply:
ok, nice, can't wait for my session to timeout :)